Tlane77/Asclepius
# Tl MVP

# Description
Added Feature to Conditions to Edit and Delete with Categories and Comments
Added Feature for Physicians to add Categories or Delete from their User Specific page.
Added Authentication for User Specific Conditions, Categories, and Comments CRUD.
#### Styling
Added 10% margins to either side of window for all views to create a whitespace border.

# Ticket Reference

> [#7 - Delete a Condition](https://github.com/nss-day-cohort-41/Asclepius/compare/tlUserProfiles/issues/7)
> [#8 - View a Conditions Comments](https://github.com/nss-day-cohort-41/Asclepius/compare/tlUserProfiles/issues/8)
> [#9 - Create a Comment](https://github.com/nss-day-cohort-41/Asclepius/compare/tlUserProfiles/issues/9)
> [#10-Delete a Comment](https://github.com/nss-day-cohort-41/Asclepius/compare/tlUserProfiles/issues/10)
>[#11-Edit a Comment](https://github.com/nss-day-cohort-41/Asclepius/compare/tlUserProfiles/issues/11)


## Type of change


# Testing Instructions
- Run both files in the SQL folder.
"tech3@health.com"
- Start program
- Go to localhost:3000 in your browser.
- Login with "tech3@example.com"
- Navigate to the "Conditions" link in the navbar.
- add a Condition
-Visually confirm Condition was added to condition List. 
-Click on the Title of Condition to Navigate to  it's details
-Visually Confirm you are on details page as logged in User.
-As a logged in user you should see Buttons for Back, Conditions, Comments , Edit and Delete.
-Edit and Delete will only appear on Active Users Conditions Details. -
-Navigate to Comments and Add a Comment to the Condition.
-Verify Visually in the comments it has been added. 
-Add Multiple Comments to a condition to test.
- Click the cancel, edit, and delete buttons as active user and ensure after forms are filled out the link brings you back to the active users page, then repeat the previous step.
- Next, click the logout button on the navbar.
- Enter tech3@health.com into the email and "password" in password field and click "Login"


# Checklist:
